### DISCUSSION TITLE
Create the Layer5.io/community for our members

### DISCUSSION THREAD

**Discussion by Aisuko** [meshery/meshery#1801] [Ideas] [answered]
As we saw the idea start from the issue comment https://github.com/layer5io/meshery/pull/1783#issuecomment-718919233, so we should do the community for our member as you can saw our community follower growing up fast. And this may a good experiences for who want to be a contributor with a startup project.

**Comment by leecalcote** [comment]
Totally, yes. With Layer5 NG soon-to-release, hopefully, we will have taken a big stride forward here - https://layer5ng.netlify.app/community/members

**Comment by leecalcote** [comment, marked as answer]
https://layer5.io/community/members was launched late last year. ✔️

**Reply by leecalcote** [reply]
https://layer5.io/community/members is up and has some recent enhancements around the ability to filter based on badges and status.

<img width="1445" alt="Screen Shot 2021-08-23 at 3 43 47 PM" src="https://user-images.githubusercontent.com/7570704/130516802-0c132bce-e85a-4653-9275-b85a9f855cf8.png">
